### ISSUE TITLE
Automatic token refresh fails if token is already expired

### ISSUE THREAD

**Issue by molikuner** [adamint/spotify-web-api-kotlin#27]
When using an already expired token the library tries to schedule for -30s. That isn't allowed for obvious reasons.

Stack:
```
Exception in thread "main" java.lang.IllegalArgumentException
	at java.util.concurrent.ScheduledThreadPoolExecutor.scheduleAtFixedRate(ScheduledThreadPoolExecutor.java:565)
	at com.adamratzman.spotify.main.SpotifyClientAPI.init(SpotifyAPI.kt:247)
	at com.adamratzman.spotify.main.SpotifyClientAPI.<init>(SpotifyAPI.kt:241)
	at com.adamratzman.spotify.main.SpotifyApiBuilder.buildClient(SpotifyAPI.kt:173)
	at com.adamratzman.spotify.main.SpotifyApiBuilder.buildClient(SpotifyAPI.kt:141)
	at com.molikuner.MainKt.main(Main.kt:28)
```

Could be fixed by refreshing the token before setting this schedule up.

**Comment by adamint** [comment]
Hmm.. the easiest solution could be to send a request before scheduling & let the api itself complain about expired tokens

**Comment by molikuner** [comment]
Fixed with #32
